docs/README.md

```
@@ -10,6 +10,8 @@
  - [dao-setup.md](dao-setup.md): (deprecated) Set up a complete Bisq DAO development environment
 - [burning-man-address-list.md](burning-man-address-list.md): Understand the versioned Burning Man DPT receiver
   allowlist
+- [burningman/filter-controlled-btc-fee-receiver-routing.md](burningman/filter-controlled-btc-fee-receiver-routing.md):
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 1; Actual: 0

(MD005, list-indent)